app/src/main/java/org/simple/clinic/contactpatient/ContactPatientUiRenderer.kt

```diff
@@ -70,6 +72,10 @@ class ContactPatientUiRenderer(
       renderPatientProfile(model.patientProfile!!, model.appointment!!)
     }
 
+    if (model.hasPatientRegistrationIDInOverdueAppointment && model.hasCurrentFacility) {
```

Comment:
When this is false, i.e., when the user doesn't have a registered facility. Should we specifically call a hide method for the label and name text? 🤔

Reply:
As this is a case that will never really happen.. Do we really need to handle this as well?

Reply:
Hmm, true. I guess I we can leave this for now, but let's keep an eye out for any potential issues related to this.